src/libgeodecomp/geometry/coord.h

```diff
@@ -117,6 +117,11 @@ class Coord<1>
         c[0] = int(other[0]);
     }
 
+    inline explicit Coord(std::initializer_list<int> list)
```

Comment:
This code is specific to C++11. Could you wrap it in preprocessor guards ("ifdef LIBGEODECOMP_WITH_CPP14")? (In LibGeoDecomp we're basically distinguishing between a C++98 compatibility mode and C++14, that's why there is no LIBGEODECOMP_WITH_CPP11).

Reply:
Same below.